templates/publisher/_beta-notification.html

```diff
@@ -0,0 +1,12 @@
+<section class="p-strip--light u-no-margin--top u-no-padding">
+  <div class="row p-beta-notification">
+    <div class="col-12">
+      <p>
+        <span class="p-muted-heading"><b>Beta</b></span>
+        Snapcraft.io is in Beta. Can't find something?
+        <a href="https://dashboard.snapcraft.io/snaps">Go back to the old version</a> or
```

Comment:
Not sure from design point of view, but maybe it would be worth making these links external.

Reply:
I agree, fixed